glossary.md

```diff
@@ -0,0 +1,220 @@
+which are derived after executing the block's transactions. These include
+various [Merkle roots][Merkle root] that notably commit to the L2 state and to
+the log events emitted during the execution.
+
+## EOA
+[EOA]: /glossary.md#EOA
+
+"Externally Owned Account", an Ethereum term to designate addresses operated by
+users, as opposed to contract addresses.
+
+## Merkle Root
+[Merkle root]: /glossary.md#merkle-roots
+
+The Merkle root is the root hash of a [Merkle Patricia tree] (MPT). A MPT is a
+sparse [trie], which is a tree-like structure that maps keys to values. The root
+hash of a MPT is a commitment to the contents of the tree, which allows a proof
+to be constructed for any key-value mapping encoded in the tree. Such a proof is
+called a Merkle proof, and can be verified against the Merkle root.
+
+## Chain Re-Organization
+[reorg]: /glossary.md#chain-re-organization
+
+A re-organization, or re-org for short, is whenever the head of a blockchain
+(its last block) changes (as dictated by the [fork choice rule]) to a block that
+is not a child of the previous head.
+
+L1 re-orgs can happen because of network conditions or attacks. L2 re-orgs are a
+consequence of L1 re-orgs, mediated via [L2 chain derivation][derivation].
+
+## Predeployed Contract ("Predeploy")
+[predeploy]: /glossary.md#predeployed-contract-predeploy
+
+A contract that was deployed on-chain at genesis (i.e. at the start of the chain).
+
+Optimistic Ethereum has the following pre-compiles:
+
+- [L1 Attributes Predeployed Contract][l1-attr-predeploy]
+
+--------------------------------------------------------------------------------
+
+# L2 Chain Concepts
+
+## L2 Chain Inception
+[L2 chain inception]: /glossary.md#L2-chain-inception
+
+The L1 block number for which the first block of the L2 chain was generated.
+
+## Rollup Node
+[rollup node]: /glossary.md#rollup-node
+
+The rollup node is responsible for [deriving the L2 chain][derivation] from the
+[L2 derivation inputs][deriv-inputs] available on L1. This is done by its
+[rollup driver] component.
+
+- cf. [Rollup Node Specification](/rollup-node.md)
+
+## Rollup Driver
+[rollup driver]: /glossary.md#rollup-driver
+
+The rollup driver is the [rollup node] component responsible for [deriving the
+L2 chain][derivation] from the [L2 derivation inputs][deriv-inputs] available on
+L1.
+
+## L2 Chain Derivation
+[derivation]:  /glossary.md#L2-chain-derivation
+
+A process that reads [L2 derivation inputs][deriv-inputs] from L1 in order to
+derive the L2 chain.
+
+cf. [L2 Chain Derivation (in Rollup Node
+Specification)](/rollup-node.md#l2-chain-derivation)
+
+## L2 Derivation Inputs
+[deriv-inputs]: /glossary.md#l2-chain-derivation-inputs
+
+This term refers to data that is found in L1 blocks and is read by the [rollup
+node] to construct [payload attributes].
+
+Chain derivation attributes include:
+- L1 block attributes
+   - block number
+   - timestamp
+   - basefee
+- [deposits]
```

Comment:
I would suggest either deleting these lines (which duplicate info on `rollup-node.md`) due to the confusion mentioned in the top-level comment, or putting something about feeds:

## Feed

This term refers to any array of data which is generated from L1 blocks by the [rollup node] in constructing [payload attributes] during block derivation.  Example feeds might be:
- a feed of L1 block attributes like `BASEFEE` and `TIMESTAMP` (computed from the L1 `blockHeader`)
- a feed of decoded deposit events (e.g. ABI decoded from events within the L1 `block.receiptsRoot`)